library/rsa.c

```diff
@@ -1480,6 +1480,10 @@ int mbedtls_rsa_rsassa_pkcs1_v15_verify( mbedtls_rsa_context *ctx,
     mbedtls_md_type_t msg_md_alg;
     const mbedtls_md_info_t *md_info;
     mbedtls_asn1_buf oid;
+    
+    /* Set first two bytes of the buffer so that code checkers will not think we are accessing uninitialized data (normally mbedtls_rsa_public or mbedtls_rsa_private would fill the buffer). values chosen are intentionally invalid*/
```

Comment:
Please wrap lines to 80 characters unless there's a very good reason not to. See https://tls.mbed.org/kb/development/mbedtls-coding-standards

Reply:
Thanks @gilles-peskine-arm I missed that. I'll update the branch with a fix shortly.